resources/guest_configs/microvm-kernel-ci-aarch64-4.14.config

```diff
@@ -2098,17 +2098,17 @@ CONFIG_STRIP_ASM_SYMS=y
 # CONFIG_READABLE_ASM is not set
 CONFIG_UNUSED_SYMBOLS=y
 # CONFIG_PAGE_OWNER is not set
-CONFIG_DEBUG_FS=y
+CONFIG_DEBUG_FS=n
 # CONFIG_HEADERS_CHECK is not set
-CONFIG_DEBUG_SECTION_MISMATCH=y
+CONFIG_DEBUG_SECTION_MISMATCH=n
 CONFIG_SECTION_MISMATCH_WARN_ONLY=y
 CONFIG_ARCH_WANT_FRAME_POINTERS=y
 CONFIG_FRAME_POINTER=y
 # CONFIG_DEBUG_FORCE_WEAK_PER_CPU is not set
 CONFIG_MAGIC_SYSRQ=y
 CONFIG_MAGIC_SYSRQ_DEFAULT_ENABLE=0x1
 CONFIG_MAGIC_SYSRQ_SERIAL=y
-CONFIG_DEBUG_KERNEL=y
+CONFIG_DEBUG_KERNEL=n
```

Comment:
Same, as before. Why do we disable debug functionality from guest kernels?

Reply:
This one actually doesn't seem to enable anything by itself. I'd say if we need it back for debugging let's add it at that point.